exchanges/poloniex/poloniex_websocket_trade.go

```diff
@@ -0,0 +1,107 @@
+package poloniex
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+)
+
+// WsCreateOrder create an order for an account.
+func (e *Exchange) WsCreateOrder(ctx context.Context, arg *PlaceOrderRequest) (*PlaceOrderResponse, error) {
+	if err := validateOrderRequest(arg); err != nil {
+		return nil, err
+	}
+	var resp []*PlaceOrderResponse
+	if err := e.SendWebsocketRequest(ctx, "createOrder", arg, &resp); err != nil {
+		return nil, err
+	}
+	if len(resp) != 1 {
+		return nil, common.ErrInvalidResponse
+	} else if resp[0].Code != 0 {
```

Comment:
In your testing did it not match documentation? If so, add comment. Otherwise, why 0?
https://api-docs.poloniex.com/spot/websocket/trade-request
> code	int	200 is ok,other is error.